docs/nix-flakes.adoc

```diff
@@ -126,23 +126,22 @@ $ nix flake new ~/.config/nixpkgs -t github:nix-community/home-manager
 2. Install Home Manager and apply the configuration by
 +
 [source,console]
-----
-$ nix build --no-link <flake-uri>#homeConfigurations.jdoe.activationPackage
-$ "$(nix path-info <flake-uri>#homeConfigurations.jdoe.activationPackage)"/activate
-----
+$ nix run <flake-uri>#homeConfigurations.jdoe.activationPackage
```

Comment:
Have you tested this? Wouldn't we need `nix shell ... -c activate`?

Reply:
I have, though not on a blank machine/VM, but my understanding is that it doesn't matter if I already have an installation.

```
nixpkgs > pwd
/home/user/.config/nixpkgs
nixpkgs > ls
flake.lock  flake.nix  home.nix
nixpkgs > nix run .#homeConfigurations.user.activationPackage
warning: Git tree '/home/user/.config/nixpkgs' is dirty
Starting Home Manager activation
Activating checkFilesChanged
Activating checkLinkTargets
Activating writeBoundary
Activating installPackages
nix profile remove /nix/store/76nsa0ljc3akw1gz7xiby4ygvj1rg396-home-manager-path
removing 'home-manager-path'
Activating linkGeneration
Cleaning up orphan links from /home/user
No change so reusing latest profile generation 205
Creating home file links in /home/user
Activating onFilesChange
Activating reloadSystemd
```

Reply:
Right, it runs `home-manager-activation` because that's the derivation's `name`